modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayBuildAdministrator.testcase

```diff
@@ -152,22 +171,18 @@ definition {
 	@priority = "5"
 	test AdminCanDeleteBuildContainsRuns {
 		task ("Given that the user add a new build with Environments Factors") {
-			Testray.goToProjectPage(projectName = "Projects_1 Project");
-
-			TestrayContextMenu.clickContextMenuProjects(name = "Routines_1 Routine");
+			ApplicationsMenu.gotoSite(site = "Testray 2");
 
-			TestrayContextMenu.selectItemFromProjectsContextMenu(item = "Select Default Environment Factors");
+			Testray.changePagination(
+				numberItems = "20",
+				valuePagination = "150");
 
-			TestrayBuild.selectDefaultEnvironmentFactors(
-				label = "Application Server",
-				label1 = "Browser",
-				listCategory = "Application Server,Browser",
-				listOption = "JBoss 6.1",
-				listOption1 = "Chrome x");
+			Testray.goToProjectPage(projectName = "Projects_1 Project");
```

Comment:
Hi @joelrocha68, I think these steps can be added in setup, as it is repeated in all tests